vcxagency-artillery/test/20-messaging/agency_logging_verify.txt

```diff
@@ -0,0 +1,392 @@
+VERIFY

```

Comment:
Was this by a chance commited by mistake? Or is there some documentation reason for including these logs?

Reply:
I used the log file to make DID verification simulation (vcxagency-artillery/test/20-messagine/art-verify-simul.yaml). I will add README.md to explain it. If you still think it's redundant, I will remove the agency_logging_verify.txt file.
Thanks.